libkarel.py

```diff
@@ -337,6 +353,21 @@ def instrucciones(self):
         """Un diccionario con el número de instrucciones que karel ejecutó"""
         return self.__instrucciones
 
+    @property
+    def longitud_stack(self):
+        """El tamanio maximo que puede tener el stack"""
+        return self.__longitud_stack
+
+    def limiteComando(self, comando):
+        """Regresa el número máximo de veces que se puede usar un comando
+
+        Si no hay límite, regresa None
+        """
+        if comando not in self.__limites:
+            return None
+
+        return int(self.__limites[comando])
```

Comment:
el arreglo `__limites` ya tiene puros ints, así que mejor puedes hacer

```suggestion
        return self.__limites.get(comando, None)
```

y quitar las dos líneas de arriba.